R/utils.R

```diff
@@ -1,13 +1,13 @@
 #' Shared Parameters
 #'
 #' @description Contains arguments that are shared between multiple functions
-#'   in the package to avoid repetition using \code{inheritParams}.
+#'   in the package to avoid repetition using `inheritParams`.
 #'
-#' @param plot_height optional, (\code{numeric}) a vector of length three with \code{c(value, min, max)}. Specifies
+#' @param plot_height (`numeric`) optional, a vector of length three with `c(value, min, max)`. Specifies
```

Comment:
```suggestion
#' @param plot_height (`numeric`) optional vector of length three with `c(value, min, max)`. Specifies
```

Reply:
Hi @m7pr,as per previous conversation where you suggested adding a comma after the option. We agreed on this suggestion, So why was the comma removed here?.https://github.com/insightsengineering/nestdevs-tasks/issues/52#issuecomment-1968800721

Reply:
We only agreed to add comma after `optional` word if the sentence does not have sense with optional at the beginning.

So: 

- `optional vector of length three` has a complete sense, so no need for a comma
- `optional specifies font size` is improper English statement - this should either be `optionally specifies font size`, or `optional, specifies font size` - hence a recommendation for commas but only in a situation where without comma it does not create proper English statement.

Does it make sense?

Reply:
okay 👍

Reply:
Since `optional` is an adjective you do not need to put comma after `optional` if it comes before the `noun`. However if it comes before the `verb` you need to have a comma. Adjectives are meant to describe nouns. For verbs you either change `optional` to `optionally` or you put comma before the verb. 

Is it more clear?

Reply:
yes, my understanding on this was while `optional` serves as a notation rather than a textual addition to the sentence. the user knows it's optional, so need to be part of description.  I am okay,  it being part of a sentence in suggested cases.